### ISSUE TITLE
Java Callback crash

### ISSUE THREAD

**Issue by petreeftime** [intel-iot-devkit/upm#308]
Clicking really fast on a button/touch sensor with an IsrCallback active in Java leads to a crash. It might happen if clicking rarely as well, but it's not as easy to replicate. It's very likely that a GlobalRef should be created for the IsrCallback object that is passed to installISR.

I added the error log and error message: [hs_err_pid335.log.txt](https://github.com/intel-iot-devkit/upm/files/40210/hs_err_pid335.log.txt)

```
Exception in thread "Thread-45" java.lang.NullPointerException: null upcall object in IsrCallback::run 
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00000000, pid=335, tid=2528844608
#
# JRE version: OpenJDK Runtime Environment (8.0) (build 1.8.0-internal-edison_2015_08_25_18_58-b00)
# Java VM: OpenJDK Server VM (25.60-b23 mixed mode linux-x86 )
# Problematic frame:
# C  0x00000000
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /home/root/hs_err_pid335.log
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
```

I am currently investigating the cause and posted this bug report as a FYI.

**Comment by petreeftime** [comment]
Was unable to replicate after reinstalling the OS on the Edison and updating MRAA to 0.8.1. Closing bug.

**Comment by petreeftime** [comment]
It still replicates. The IsrCallback can be cleaned up by the GC if it goes out of scope in Java, C only saves a weak reference to the object. SWIG_DIRECTOR_OWNED(IsrCallback) is required to be added after %feature ("director") IsrCallback.

**Comment by petreeftime** [comment]
Fixed in https://github.com/intel-iot-devkit/upm/pull/313
